applications/ConvectionDiffusionApplication/custom_elements/laplacian_embedded_element.cpp

```diff
@@ -0,0 +1,433 @@
+// KRATOS ___ ___  _  ___   __   ___ ___ ___ ___
+//       / __/ _ \| \| \ \ / /__|   \_ _| __| __|
+//      | (_| (_) | .` |\ V /___| |) | || _|| _|
+//       \___\___/|_|\_| \_/    |___/___|_| |_|  APPLICATION
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//                   Franziska Wahl
```

Comment:
```suggestion
//  Main authors:    Franziska Wahl
//
```
I think you did everything in here.